e2e/whisper/whisper_jail_test.go

```diff
@@ -48,46 +41,33 @@ type WhisperJailTestSuite struct {
 func (s *WhisperJailTestSuite) StartTestBackend(opts ...e2e.TestNodeOption) {
 	s.BackendTestSuite.StartTestBackend(opts...)
 
+	s.Timeout = time.Minute * 5
 	s.WhisperAPI = whisper.NewPublicWhisperAPI(s.WhisperService())
 	s.Jail = s.Backend.JailManager()
 	s.NotNil(s.Jail)
+
 	s.Jail.BaseJS(baseStatusJSCode)
 }
 
-func (s *WhisperJailTestSuite) GetAccountKey(account struct {
-	Address  string
-	Password string
-}) (*keystore.Key, string, error) {
+func (s *WhisperJailTestSuite) AddKeyPair(address, password string) (string, error) {
```

Comment:
I like so far the idea of returning the `Hash ID` for the account which then is used to retrieve the public key easily. It really simplifies alot of ugly code around. I do wonder if we can add that to `AccountManager` as well, to simplify the code to get a accounts `PublicKey` hash using the previous boiler plate.

Reply:
> I like so far the idea of returning the Hash ID for the account which then is used to retrieve the public key easily.

This is a Whisper feature. When you add a key pair to Whisper, it returns a random ID that can be used to get public or private key. This ID is random in go-ethereum implementation, but in our fork, it's a hex-encoded public key.

>  I do wonder if we can add that to AccountManager as well, to simplify the code to get a accounts PublicKey hash using the previous boiler plate.

Do you know any use-case where we need a public key of an account and it requires a lot of boilerplate? In this case, I believe that the previous `GetAccountKey` test suite method was added before Whisper V5 which greatly improved its public API.

Reply:
In previous code like https://github.com/status-im/status-go/blob/3d46fb91348af7121627f0f8f272ea0c19f1c6f3/geth/api/backend_accounts_test.go#L68-L83, we had to do a messy calls to get the `hash` public key, am not sure how much the API has changed, but if we can keep things simplified and have such similar ops kept into methods as needed, I think this would be good.

But no code is better than more code, am don't think that single case would solidify enough for such, so we can just keep on note and decide as things evolve.

Reply:
This could be simplified, however, I can't find this code after moving e2e tests out.

Reply:
Must have being removed, then no need to bother then.